@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -612,9 +639,16 @@ export class TileGeometryCreator {
                             const unitFactor =
                                 metricUnits === "Pixel" ? mapView.pixelToWorld * 0.5 : 1.0;
 
+                            lineMaterial.color.set(
+                                getOptionValue(
+                                    getPropertyValue(technique.color, mapView.zoomLevel),
+                                    "#000000"
+                                )
+                            );
+
                             lineMaterial.lineWidth =
                                 getOptionValue(
-                                    getPropertyValue(technique.lineWidth, zoomLevel),
+                                    getPropertyValue(technique.lineWidth, mapView.zoomLevel),
```

Comment:
... here (look at above comment).

Reply:
this is the 2nd case of my previous comment. We need up-to-date value here, and can't use cached variable;

Reply:
Got it!